code/modules/mod/modules/modules_supply.dm

```diff
@@ -79,7 +78,6 @@
 		dropped_crate.forceMove(target_turf)
 		balloon_alert(mod.wearer, "dropped [dropped_crate]")
 		drain_power(use_power_cost)
-		mod.wearer.update_inv_back()
```

Comment:
Were these redundant?

Reply:
Fikou told me in one of their reviews to remove it, so I suppose so.